django/db/models/fields/related.py

```diff
@@ -89,6 +89,18 @@ class RelatedField(FieldCacheMixin, Field):
     many_to_many = False
     many_to_one = False
 
+    def __init__(
+        self,
+        related_name=None,
+        related_query_name=None,
+        limit_choices_to=None,
+        **kwargs
```

Comment:
```suggestion
        **kwargs,
```

Reply:
Duly noted, thanks 👌 

I wasn't aware Django is pushing in this direction for indentation style (related to `black`'s style I guess) - I always thought making `flake8` happy was enough.

Reply:
It's not really related with `black`. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).